src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -275,12 +275,15 @@ Graph QuantizeGraph(Graph &&src) {
   std::unordered_map<Node*, NodePtr> mirror_map;
   std::unordered_map<NodePtr, NodePtr> reverse_mirror_map;
   nnvm::NodeEntryMap<NodeEntry> mirror_entry_map;
+  static int verbose = dmlc::GetEnv("MXNET_QUANTIZATION_VERBOSE", 1);
```

Comment:
Please use default = 0, in script you can set to 1 in non-quiet mode

Reply:
I suggest in script you can set to 0 in non-quiet mode.